com.unity.mobile.notifications/Editor/NotificationSettings.cs

```diff
@@ -117,15 +117,15 @@ public static bool RequestAuthorizationOnAppLaunch
             /// <summary>
             /// Configure the notification interaction types your app will include in the authorisation request if RequestAuthorizationOnAppLaunch is enabled. Alternatively you can specify them when creating a `AuthorizationRequest` from a script.
             /// </summary>
-            public static PresentationOption DefaultAuthorizationOptions
+            public static AuthorizationOption DefaultAuthorizationOptions
```

Comment:
Isn't this a a breaking change? Shouldn't you have backwards compatible API present with Obsolete attribute?

Reply:
Unfortunately it is. I think it was originally a typo no one noticed, since the two enums have very similar names and back then they even had the same values.
My reasoning for breaking was:
- No one complained about this so probably no one used it.
- Any serialized value is serialized as int, so deserializing will result in correct values.
- This is Editor only API for manipulating settings, for which UI exists too. So someone uses it, it's probably like one place in the code and will be easy to fix.
- Obsoleting/deprecating would mean introducing new API and inventing a name for it, since the most natural name is already used.
So considering all the points above, I decided that this API breakage is the best course of action.